strr-examiner-web/app/components/Host/Expansion/FilingHistory.vue

```diff
@@ -0,0 +1,93 @@
+<script setup lang="ts">
+
+defineEmits<{ close: [void] }>()
+
+const { getApplicationFilingHistory } = useExaminerStore()
+const { activeHeader } = storeToRefs(useExaminerStore())
+
+// business requirement: don’t show auto approval logic events - it’s implied that it’s done by the system
+const HIDDEN_EVENTS: FilingHistoryEventName[] = [
+  FilingHistoryEventName.AUTO_APPROVAL_FULL_REVIEW,
+  FilingHistoryEventName.AUTO_APPROVAL_PROVISIONAL,
+  FilingHistoryEventName.AUTO_APPROVAL_APPROVED
+]
+
+const { data: filingHistory, status } = await useLazyAsyncData<FilingHistoryEvent[]>(
```

Comment:
If registration events need to be shown in `RegistrationInfoHeader` then this would need to be updated?

Reply:
I was able to see the history for Registration using the application number. You think the Reg history could be different?

Reply:
Reg history might have be cancelled, etc?

Reply:
As discussed, I will add a Reg History.

Reply:
Added Reg History